example/harness.go

```diff
@@ -98,12 +98,60 @@ func mustLogin(in http.Handler) http.Handler {
 	})
 }
 
-func Mount(mux *http.ServeMux) {
+func Mount(mux *http.ServeMux, l log.Logger) {
 	a := account{}
 	s := search{}
 	au := auth{}
 
-	prottp.Handle(mux, a, mustLogin)
-	prottp.Handle(mux, au, server.WithHeader)
-	prottp.Handle(mux, s)
+	var sinterceptor = trace.UnaryServerInterceptor(l, []trace.MethodTrace{
+		{
+			MethodName: "Search",
+
+			RequestFields:  []string{".Query", ".PageNumber", "ResultPerPage"},
```

Comment:
Is it intentional that `.Query` and `.PageNumber` start with a dot (`.`), and `ResultPerPage` doesn't?

The log result:

```
level=info ts=2018-11-16T09:52:59.609712851+11:00 caller=trace.go:83 req_id=9e272c18-c1f0-424e-a0f0-5f713761c7d6 Request.Query="\"query string\"" Request.PageNumber=1 RequestResultPerPage=10 Response.[0]="<err no such field.>" Response.Result.[0]="example.Result{Url:\"query string\", Title:\"\", Snippets:\"\", SomeSnakedName:2}"
```

And the odd part (note the key)

```
Request.Query="\"query string\""
```

vs 

```
RequestResultPerPage=10
```

Reply:
changed.